docs/question/server.md

```diff
@@ -83,15 +83,15 @@ For generated floating point answers, it's important to use consistent rounding
 
 ## Step 2: `prepare`
 
-Next, the `prepare` function is called after all elements (e.g. `<pl-number-input>`) have run `prepare()`. This is typically done to do any sort of final post-processing, but is not commonly used.
+Next, the `prepare` function is called after all elements (e.g. `<pl-integer-input>`) have run `prepare()`. This is may be done to do any sort of final post-processing, but is not commonly used.
```

Comment:
```suggestion
Next, the `prepare` function is called after all elements (e.g. `<pl-integer-input>`) have run `prepare()`. This may be done to do any sort of final post-processing, but is not commonly used.
```